### PR TITLE
better explain what code disctionary contains

### PR DESCRIPTION
**Pull request by matkoniecz**
https://github.com/codespell-project/codespell/pull/2144#issuecomment-966919948

### CODE REVIEW

File: codespell_lib/_codespell.py [hunk 1]
```diff
@@ -58,7 +58,7 @@
         True, True, supported_languages_en, supported_languages_en),
     ('usage', 'for replacing phrasing with recommended terms', '_usage',
         None, None, None, None),
-    ('code', 'for words common to code and/or mathematics that might be typos', '_code',  # noqa: E501
+    ('code', 'for words from code and/or mathematics that are likely typos in other context (such as uint)', '_code',  # noqa: E501
```
Comment (peternewman) [inline codespell_lib/_codespell.py:61]: s/context/contexts/

Thanks for this, I'm not sure if the example helps, as it's not necessarily a very strong example, and opens us up to the challenge of our typos appearing in our codebase if we wanted to check with the code dictionary.
```suggestion
    ('code', 'for words from code and/or mathematics that are likely to be typos in other contexts', '_code',  # noqa: E501
```

**Review by peternewman — CHANGES_REQUESTED** [review]
I was going to ask you to add disctionary but I see it's already in the dictionary. :laughing:

**Comment by matkoniecz** [comment]
> I was going to ask you to add disctionary but I see it's already in the dictionary. 😆

Well, let's say that there is a reason why I am interesting in typo detection

**Review by peternewman — APPROVED** [review]
LGTM thanks!

**Pull request merged**